cli/USER_GUIDE.md

```
@@ -52,7 +53,7 @@ Key features:
 | Python 3.11+ | Runtime | [python.org](https://www.python.org/downloads/) |
 | Docker | Container execution | [Docker Desktop](https://www.docker.com/products/docker-desktop/) |
 | yq | YAML parsing | `choco install yq` / `brew install yq` / `apt install yq` |
-| act | Local GitHub Actions | `choco install act-cli` / `brew install act` |
+| act | Local GitHub Actions | `choco install act-cli` / `brew install act` / `curl -s https://raw.githubusercontent.com/nektos/act/master/install.sh | sudo bash` |
```

markdownlint-cli2 (0.21.0)

[warning] Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)